pkg/project/project.go

```diff
@@ -0,0 +1,39 @@
+package project
+
+import (
+	"context"
+	"os"
+
+	"golang.org/x/oauth2/google"
+)
+
+// FromEnv attempts to find active project id from environment variables.
+// The environment variables are taken from this page: https://cloud.google.com/functions/docs/configuring/env-var
+// If no project found, returns empty string.
+func FromEnv() string {
+	envs := []string{
+		"GOOGLE_CLOUD_PROJECT",
```

Comment:
Did you test those on your machine?
I think those env are set in runtime environments but it is not necessary will appear in the your local env (for example, on my machine those are not set but I do have an active gcloud project)

Reply:
Yes, it's only for run times, I plan to delete this function and keep only FromApplicationDefault